cmd/limactl/network.go

```diff
@@ -174,23 +174,23 @@ func networkCreateAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	switch mode {
-	case networks.ModeBridged:
+	if mode == networks.ModeBridged {
```

Comment:
I don't think you need to change the structure of the code to fix this issue. Keep the switch.